app/design/frontend/base/default/layout/oyst_oneclick.xml

```diff
@@ -14,6 +14,23 @@
 
     <default>
         <block type="oyst_oneclick/oneClick" name="oyst.oneclick.express.checkout" template="oyst/oneclick/oneclick.phtml"/>
+        <reference name="head">
+            <block type="core/text" name="oyst-oneclick-metadata">
+                <action method="setText" ifconfig="oyst_oneclick/oneclick/tracking">
+                    <text>
+                        <![CDATA[
+                            <meta name="oyst_tracker_info_plugin_version" content="1.9.4">
+                            <meta name="oyst_tracker_info_freepay_activated" content="1">
+                            <meta name="oyst_tracker_info_freepay_apikey" content="1">
```

Comment:
content="1" has to be binded to freepay apikey field state. 1 = fill ; 0 = empty